src/applications/mhv/landing-page/utilities/index.js

```diff
@@ -0,0 +1,11 @@
+const getHostname = href => {
+  let result;
+  try {
+    result = new URL(href).hostname;
```

Comment:
This would fail on relative URLs, regardless of which domain the code is running in, e.g. `new URL('/health-care/schedule-view-va-appointments/').hostname` throws an error, so this function. I think our data contains a mix of absolute and relative URLs, so I _think_ we will end up with 'localhost' values in unexpected places


Also unsure if we need to manually record the hostname, per my other comment.

Reply:
That was the purpose of the try...catch block.